walkthrough/index.html

```diff
@@ -129,17 +132,14 @@
 - [[Yes-&gt;Linux path]]
 - [[No-&gt;Talk to a Volunteer]]</tw-passagedata><tw-passagedata pid="8" name="Linux Introduction" tags="" position=""># Installing Exercism on Linux
 
-To install Exercism on Linux you need to get the `exercism`
-executable; different process architectures have different
-`exercism` executables, so you need to install the
-right exectuable matching your process architecture.
+You can use the [snap package](https://snapcraft.io/exercism) to install the command-line client, or install it manually. If you are unsure which one to choose, try using snap first. More information is available in the next section.
 
 ---
 
-## Do you know your process architecture (x86 (32-bit) vs x86_64 (64-bit) vs ARM)?
+## I would like to install the CLI
 
-- [[Yes-&gt;Linux tarball]]
-- [[No-&gt;Linux architecture]]</tw-passagedata><tw-passagedata pid="9" name="Linux path" tags="" position=""># Adding `~/bin` to `$PATH` in Bash
+- [[Using snap-&gt;Linux - Snap]]
+- [[Using the source code-&gt;Linux tarball]]</tw-passagedata><tw-passagedata pid="9" name="Linux path" tags="" position=""># Adding `~/bin` to `$PATH` in Bash
 
 Note: If you’re not running Bash try to adjust the
```

Comment:
Change `you're` to `you are`

Reply:
I fixed all of the contractions in the file.